web/components/receiptForm.tsx

```diff
@@ -31,11 +32,69 @@ export const ReceiptForm = () => {
           </DrawerHeader>
           <div className="p-4">
             <form className="flex flex-col gap-4">
-              <Input required type="number" label="Subtotal" startContent="$" />
-              <Input required type="number" label="GST/HST" startContent="$" />
-              <Input type="number" label="Gratuity" startContent="$" />
-              <Input required type="date" label="Date" />
-              <Input disabled type="number" label="Total" />
+              <div className="flex items-center justify-between">
+                <div className="flex items-center">
+                  <label className="text-sm">Subtotal</label>
+                  <Tooltip content="The pre-tax ammount">
+                    <CircleHelp className="w-4 h-4 ml-2" />
+                  </Tooltip>
+                </div>
+                <Input
+                  required
+                  type="number"
+                  startContent="$"
+                  placeholder="0.00"
+                  className="max-w-52"
+                />
+              </div>
+              <div className="flex items-center justify-between">
+                <div className="flex items-center">
+                  <label className="text-sm">GST/HST</label>
+                  <Tooltip content="Goods and services tax/harmonized sales tax">
+                    <CircleHelp className="w-4 h-4 ml-2" />
+                  </Tooltip>
+                </div>
+                <Input
+                  required
+                  type="number"
+                  startContent="$"
+                  placeholder="0.00"
+                  className="max-w-52"
+                />
+              </div>
+              <div className="flex items-center justify-between">
+                <div className="flex items-center">
+                  <label className="text-sm">Gratuity</label>
+                  <Tooltip content="The gratuity provided on this expense">
```

Comment:
```suggestion
                  <Tooltip content="The gratuity listed on this receipt.">
```

Reply:
Might be good to keep our language consistent. I've been defining an expense as an individual item on a receipt